pkg/controller/migcluster/migcluster_controller.go

```diff
@@ -76,17 +89,76 @@ type ReconcileMigCluster struct {
 // +kubebuilder:rbac:groups=migration.openshift.io,resources=migclusters,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=migration.openshift.io,resources=migclusters/status,verbs=get;update;patch
 func (r *ReconcileMigCluster) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	log.Info(fmt.Sprintf("[mCluster] RECONCILE [nsName=%s/%s]", request.Namespace, request.Name))
+
 	// Fetch the MigCluster instance
 	instance := &migrationv1alpha1.MigCluster{}
```

Comment:
I know this was named by a code generator but `instance` is vague and leads to awkward naming elsewhere in the code.  Perhaps `migCluster` would be clearer.